src/utils/BatchManager.js

```diff
@@ -2,8 +2,13 @@ function errorToSerializable(error) {
   // istanbul ignore next
   if (error === undefined) throw new TypeError('No error was passed');
 
-  // make sure it is an error object so we can serialize it properly
-  const err = error instanceof Error ? error : new Error(error);
+  // make sure it is an object that is Error-like so we can serialize it properly
+  // if it's not an actual error then we won't create an Error so that there is no stack trace
+  // because no stack trace is better than a stack trace that is generated here.
+  const err = typeof error === 'object' && typeof error.stack === 'string'
```

Comment:
If you really want that, you could set `Symbol.toStringTag` to `"Error"` - but I don't think it's a good idea to allow pojos. Error stacks will be standardized sooner than later, and I'd think we want proper stack traces.